<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Student</class>
 <widget class="QDialog" name="Add_Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>491</width>
     <height>661</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>570</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确定添加</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NamelineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>学号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>电话</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NumberlineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TellineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="YuanxilineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>院系</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>专业</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZhuanyelineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>610</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>寝室</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="QinshilineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ClasslineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>490</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>班级</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SexlineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>性别</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordlineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗圆宋简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
